api/rpc/client/client.go

```diff
@@ -18,18 +18,11 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/state"
 )
 
-// TODO: this duplication of strings many times across the codebase can be avoided with issue #1176
-var client Client
-var Modules = map[string]interface{}{
-	"share":  &client.Share.Internal,
-	"state":  &client.State.Internal,
-	"header": &client.Header.Internal,
-	"fraud":  &client.Fraud.Internal,
-	"das":    &client.DAS.Internal,
-	"p2p":    &client.P2P.Internal,
-	"node":   &client.Node.Internal,
-	"blob":   &client.Blob.Internal,
-}
+var (
+	// staticClient is used for generating the OpenRPC spec.
+	staticClient Client
+	Modules      = moduleMap(&staticClient)
```

Comment:
nit: exported should be listed first, white space between

Reply:
not blocking so merging